tvb_framework/tvb/interfaces/web/controllers/base_controller.py

```diff
@@ -60,6 +60,7 @@ class BaseController(object):
     """
     This class contains the methods served at the root of the Web site.
     """
+    MAX_SIZE_ERROR_MSG = "Max operation size has been exceeded"
```

Comment:
`MAX_SIZE_ERROR_MSG = "Max operation size has been exceeded. The current project admin can change this limit in Project - Basic Properties"` ?